wadm/mix.exs

```diff
@@ -24,7 +24,12 @@ defmodule Wadm.MixProject do
       {:yaml_elixir, "~> 2.8"},
       {:cloudevents, "~> 0.4.0"},
       {:uuid, "~> 1.1"},
-      {:lattice_observer, git: "https://github.com/wasmcloud/lattice-observer"},
+      {:libcluster, "~> 3.3"},
+      {:horde, "~> 0.8.5"},
+      {:gnat, "~> 1.4"},
+      {:redix, "~> 1.1"},
+      {:lattice_observer, path: "../../lattice-observer"},
+      # {:lattice_observer, git: "https://github.com/wasmcloud/lattice-observer"},
```

Comment:
Does this need to be uncommented before merge?

Reply:
I'll get switched back to git once the lattice-observer pending PR is in main

Reply:
But that can happen in another PR